lib/src/chart/base/axis_chart/axis_chart_helper.dart

```diff
@@ -51,4 +53,52 @@ class AxisChartHelper {
       yield max;
     }
   }
+
+  /// Calculate translate offset to keep [SideTitle] child
+  /// placed inside its corresponding axis.
+  /// The offset will translate the child to the closest edge inside
+  /// of the corresponding axis bounding box
+  Offset calcFitInsideOffset({
+    required AxisSide axisSide,
+    required double? childSize,
+    required double parentAxisSize,
+    required double axisPosition,
+    required double distanceFromEdge,
+  }) {
+    if (childSize == null) return Offset.zero;
+
+    // Find title alignment along its axis
+    final axisMid = parentAxisSize / 2;
+    final mainAxisAligment = (axisPosition - axisMid).isNegative
```

Comment:
There is a typo issue in `mainAxisAligment`. Please fix it.

Reply:
I have fix the typo in the latest commit